specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/ManagedEnvironmentsMaintenanceConfigurations.json

```diff
@@ -65,7 +65,7 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/managedEnvironment/{environmentName}/maintenanceConfigurations/{configName}": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/managedEnvironments/{environmentName}/maintenanceConfigurations/{configName}": {
```

Comment:
>managedEnvironments [](http://example.com/codeflow?start=96&length=19)

Please confirm if the API with managedEnvironment shipped  already ? 

Or if its not and this is just a swagger change and there is no impact ?

Reply:
It has already been shipped. Refer https://github.com/Azure/azure-rest-api-specs/blob/fe44d3261ff0ea816315126120672ccec78c3074/specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/ManagedEnvironments.json#L18C62-L18C81